pkgs/test/cross/default.nix

```diff
@@ -146,6 +146,8 @@ let
     pkgs.pkgsCross.mips64el-linux-gnuabi64.stdenv
     pkgs.pkgsCross.mips64el-linux-gnuabin32.stdenv
     pkgs.pkgsCross.mingwW64.stdenv
+    # Unlike later versions, cross-compiled 8.10.7 can reliably bootstrap a native build
+    pkgs.pkgsCross.riscv64.haskell.compiler.integer-simple.ghc8107
```

Comment:
Maybe an idea, I think we can expand the test list a bit (in release-haskell.nix), but should keep it to the essential stuff in the test.

The nice thing about 8.10.7 is of course that it can be used to start a bootstrap chain of all compilers in nixpkgs.

Also, we should be able to use gmp now if gmp can be cross compiled to riscv64.